app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -41,21 +41,27 @@ OneMediaType.propTypes = {
   linkUrl: PropTypes.string.isRequired,
 };
 
-const TweetBody = ({ text, mediaUrl, linkUrl }) => {
-  if (mediaUrl === null && linkUrl === null) {
-    return <TweetText text={text} />;
+const removeRT = text => {
+  return text.substr(text.indexOf(' ') + 1);
```

Comment:
Does this mean `removeRetweet`? Prefer avoiding acronyms over full terms in functions

Is the text before the space always dynamic? Could this be a little more explicit?